Created network.md file for network related information

srajamohamed opened the pull request:
Created a new file network.md to incorporate all the network related config information related to AMS.
Also, updated the page navigation on all the pages and included page navigation at the bottom as well.

Anbanerj reviewed (commented):
Had a discussion with Syed for some minor changes.

srajamohamed commented:
Hi Anjan,
Incorporated all the changes and updated the network.md file with image. Also, updated the availability image including the support for regions in India.
